CITATION.cff

```diff
@@ -0,0 +1,28 @@
+cff-version: 1.2.0
+title: 'TSV Data Analytics'
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: 
+    family-names: CrowdStrike
+    email: falconpy@crowdstrike.com
```

Comment:
This needs to be updated. Currently I am using amit.jaiswal@gmail.com as the author email id.